docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,41 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Input
+
+<p class="description">The `Input` component allows users to enter and edit text.</p>
+
+## Basic input
+
+```js
+import InputUnstyled from '@mui/base/InputUnstyled';
+```
+
+{{"demo": "UnstyledInputBasic.js", "defaultCodeOpen": false}}
+
+## Adornments
```

Comment:
Should the adornment props be listed in this section?

Reply:
Good point, I've changed a bit the wording with https://github.com/mui/material-ui/pull/31881/commits/c46d7b3996ee48a274d585abb6d79857f11aaf27